### PR TITLE
donate: add Icons8

### PR DESCRIPTION
**Pull request by Barthalion**

### CODE REVIEW

File: templates/public/donate.html [hunk 1]
```diff
@@ -53,6 +53,15 @@
     <img width="496" height="76" src="{% static "pia_logo.png" %}"
     class="sponsor-btn-pia" title="" alt="Private Internet Access logo"/></a>
 
+    <p>We would also like to thank <a href="https://icons8.com/" title="Icons8">
+    Icons8</a> for their yearly monetary donation. Icons8 is a team of talented
+    designers and engineers working on fullfilling all your icons needs.</p>
```
Comment (polyzen) [inline templates/public/donate.html]: fulfilling*

Comment (Barthalion) [reply templates/public/donate.html]: There is no such thing as too many l's!

Thanks, will fix it.

**Comment by jelly** [comment]
Looks good to me 👍

File: templates/public/donate.html (hunk 1 above)
Comment (Barthalion) [reply templates/public/donate.html]: Done.

**Pull request merged**